### ISSUE TITLE
Couldn't catch error with graphql-ruby@1.9's new GraphQL::Execution::Interpreter

### ISSUE THREAD

**Issue by layerssss** [exAspArk/graphql-errors#20]
I'm setting up graphql-ruby and graphql-errors on an old project. (rails 4.2). I opted in the new engine introduced in graphql-ruby@1.9:

https://github.com/rmosolgo/graphql-ruby/blob/master/guides/queries/interpreter.md

I wasn't able to catch error with `rescue_from RuntimeError` in config, and a `raise 'TODO'` in the resolver. The error results in caught by other error middleware(`better_errors` gem).

I discovered commenting out the new graphql-ruby engine does fix this.

```
# Opt in to the new runtime (default in future graphql-ruby versions)
# use GraphQL::Execution::Interpreter
# use GraphQL::Analysis::AST
```

The new engine would become the default in future as the doc says.

**Comment by exAspArk** [comment]
Hey @layerssss,

This issue is related to https://github.com/exAspArk/graphql-errors/issues/17. 

TL;DR I won't be adding support for the new `Interpreter` as long as the `graphql` gem has this feature in the experimental status:

![image](https://user-images.githubusercontent.com/577441/74447754-dfa50580-4e47-11ea-9ea9-17861c942dfb.png)

I would like to avoid making unnecessary breaking changes from one `graphql` gem version to another. But feel free to open a PR if you want to contribute and help other people :)

**Comment by layerssss** [comment]
@exAspArk Thanks for the clarification. I'll disable the `Interpreter` for now and keep an eye on the space.
